experiments/views.py

```diff
@@ -319,3 +319,20 @@ def get(self, request, experiment, file_format='csv', data_point=None):
             return create_file_response_single(file_format, qs.first())
         else:
             return create_download_response_zip(file_format, self.experiment)
+
+
+class DownloadFormView(UserAllowedMixin, generic.FormView):
+    form_class = DownloadForm
+    template_name = 'experiments/download.html'
+
+    def get_form_kwargs(self):
+        kwargs = super().get_form_kwargs()
+        kwargs['experiment'] = self.experiment
+        return kwargs
+
+    def form_valid(self, form):
+        file_format = form.cleaned_data['file_format']
+        queryset = self.experiment.datapoint_set\
+            .filter(session__experiment_state__in=form.cleaned_data['include_status'])\
```

Comment:
More style related, for ease of viewing I'd say a space before the slash is preferable